### DISCUSSION TITLE
post/yubikey-restore/

### DISCUSSION THREAD

**Discussion by giscus** [Yesterday17/discussion#3] [Blog]
# post/yubikey-restore/

最近丢了一把 Yubikey，但又找回来了，啼嘿

https://blog.mmf.moe/post/yubikey-restore/

**Comment by userid** [comment]
硬件密钥实际上是没有backup机制的，所以用来做登录凭证其实风险不小

**Reply by Yesterday17** [reply]
是的，所以首先要有多把（防止一把丢了导致登录不上去），或者结合其他认证手段（比如OTP）。

如果这些手段都做了的话，那丢了之后只要revoke一遍所有网站上的认证就可以了。相比之下，非硬件密钥如果泄露，也需要修改所有网站上的密码/MFA，这么看来好像也差不多 :thinking:
